### ISSUE TITLE
Unable to build prod version

### ISSUE THREAD

**Issue by langecode** [kirbydesign/designsystem#3]
It seems the current setup is unable to build with production environment. Running
```bash
ng build --prod
```
Gives me

```
Date: 2018-09-09T09:25:43.654Z
Hash: 4ed63ffac5b26e1b5f4f
Time: 4082ms
chunk {0} runtime.a66f828dca56eeb90e02.js (runtime) 1.05 kB [entry] [rendered]
chunk {1} styles.82fe6436242c28f4f6f0.css (styles) 148 bytes [initial] [rendered]
chunk {2} polyfills.cdf87a8e5b31fe8a11f1.js (polyfills) 130 bytes [initial] [rendered]
chunk {3} main.a2bc6cab25d0aa41c17a.js (main) 128 bytes [initial] [rendered]

ERROR in src/app/card/card.component.html(3,7): : Property 'subtitle' does not exist on type 'CardComponent'.
```

Removing `--prod` it runs through though.

I haven't dug into it yet. @jakobe, @Anders-Bloch, any ideas?

**Comment by jakobe** [comment]
Hi Thor, that's my bad, it was just a quick and dirty component to start with something, so I added the subtitle to the template, but not the component class (works in debug, but fails on prod build). It's fixed now, so try again. J :)

**Comment by jakobe** [comment]
![Works on my machine](https://rlv.zcache.ca/works_on_my_machine_classic_round_sticker-r56ce1cc314be46efbe749e9c58c761d5_v9waf_8byvr_540.jpg)

**Comment by langecode** [comment]
Solved by commit 64858e744a7b46d5c0f4fca254c29cceaa476ff2
